test/functional/test_runner.py

```diff
@@ -246,6 +246,7 @@
     'rpc_help.py',
     'feature_help.py',
     'feature_shutdown.py',
+    'p2p_ibd.py',
```

Comment:
This is a confusing name because it doesn't really test IBD (the main purpose of which is to sync the tip).
Maybe 'p2p_tx_relay_while_ibd.py'

Reply:
p2p_feefilter_ibd ?

Reply:
p2p_ibd_txrelay?